### PR TITLE
52354 Profile and add animation from template

### PR DESCRIPTION
**Pull request by diemntn-0165**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52354)

## WHAT (optional)
- Show profile and add animation from template
## HOW
- Install Bootstrap with Webpack with Rails
## Evidence (Screenshot or Video)
Screenshot from 2022-08-18 09-36-11
Screenshot from 2022-08-18 09-38-13

### CODE REVIEW

**Review by fonqhoanq — APPROVED** [review]

**Comment by diemntn-0165** [comment]
ready

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <!-- top footer statrs -->
     <div class="row top-footer">
         <div class="col-xs-12 col-sm-3 footer-logo-block color-gray">
-            <a href="#"> <img src="assets/food-picky-logo.png" alt="Footer logo"> </a> <span>Order Delivery &amp; Take-Out </span> 
+          <%= link_to image_tag("food-picky-logo.png"), root_path%> <span>Order Delivery &amp; Take-Out </span>
```
Comment (manhnt-0960) [inline app/views/layouts/_footer.html.erb]:
```suggestion
          <%= link_to image_tag("food-picky-logo.png"), root_path %> <span>Order Delivery &amp; Take-Out </span> 
```

Comment (manhnt-0960) [reply app/views/layouts/_footer.html.erb]: i18n e

**Comment by diemntn-0165** [comment]
ready

**Pull request merged**